spyder/plugins/projects/widgets/main_widget.py

```diff
@@ -598,6 +600,151 @@ def show_widget(self):
         self.raise_()
         self.update()
 
+    def handle_switcher_modes(self, mode):
+        """
+        Populate switcher with files in active project.
```

Comment:
Please apply to this docstring, and the ones of the new methods below, the small suggestions I posted above.